public/app/plugins/panel/canvas/CanvasPanel.tsx

```diff
@@ -225,6 +226,11 @@ export class CanvasPanel extends Component<Props, State> {
     isSetBackgroundOpen = true;
   };
 
+  tooltipCallback = (tooltip: CanvasTooltipPayload | undefined) => {
+    this.scene.tooltip = tooltip;
+    this.forceUpdate();
```

Comment:
Is this `forceUpdate` necessary?

Reply:
yes - it's for rendering/showing up the tooltip..